config/locales/views.fr.yml

```
@@ -231,6 +231,15 @@ fr:
           one: 'Une analyse mise à jour :'
           other: "%{count} analyses mises à jour :"
           zero: Aucune analyse mise à jour.
+    company_mailer:
+      confirmation_solicitation:
+        content_html: "<p>Votre demande a bien été prise en compte sur le site Place des Entreprises. Si un complément d’informations est nécessaire, nous revenons vers vous dans les 72 heures.</p><p>Vous serez ensuite contacté par les partenaires publics susceptibles de vous aider.</p><p>L’Equipe Place des Entreprises assure un suivi qualité pour qu’aucune demande ne reste sans réponse.</p><p>Merci d’avoir utilisé notre service.</p><p>Bonne journée,</p>"
```

“Équipe” avec une minuscule plutôt qu’une majuscule ? En tout cas, avec un accent :)
```suggestion
        content_html: "<p>Votre demande a bien été prise en compte sur le site Place des Entreprises. Si un complément d’informations est nécessaire, nous revenons vers vous dans les 72 heures.</p><p>Vous serez ensuite contacté par les partenaires publics susceptibles de vous aider.</p><p>L’équipe Place des Entreprises assure un suivi qualité pour qu’aucune demande ne reste sans réponse.</p><p>Merci d’avoir utilisé notre service.</p><p>Bonne journée,</p>"
```

---

la minuscule oui, mais on met un accent après un apostrophe ?

---

apparemment oui, j'ai des vieux souvenirs de primaire qui me disaient que non